<!DOCTYPE html>
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <title>The source code</title>
  <link href="../resources/prettify/prettify.css" type="text/css" rel="stylesheet" />
  <script type="text/javascript" src="../resources/prettify/prettify.js"></script>
  <style type="text/css">
    .highlight { display: block; background-color: #ddd; }
  </style>
  <script type="text/javascript">
    function highlight() {
      document.getElementById(location.hash.replace(/#/, "")).className = "highlight";
    }
  </script>
</head>
<body onload="prettyPrint(); highlight();">
  <pre class="prettyprint lang-js"><span id='org-json-JSONException'>/**@class org.json.JSONException
</span>@extends java.lang.Exception

 Thrown to indicate a problem with the JSON API. Such problems include:
 &lt;ul&gt;
   &lt;li&gt;Attempts to parse or construct malformed documents
   &lt;li&gt;Use of null as a name
   &lt;li&gt;Use of numeric types not available to JSON, such as {@link Double#isNaN() NaNs} or {@link Double#isInfinite() infinities}.
   &lt;li&gt;Lookups using an out of range index or nonexistent name
   &lt;li&gt;Type mismatches on lookups
 &lt;/ul&gt;

 &lt;p&gt;Although this is a checked exception, it is rarely recoverable. Most
 callers should simply wrap this exception in an unchecked exception and
 rethrow:
 &lt;pre&gt;  public JSONArray toJSONObject() {
     try {
         JSONObject result = new JSONObject();
         ...
     } catch (JSONException e) {
         throw new RuntimeException(e);
     }
 }&lt;/pre&gt;
*/
var JSONException = {


};</pre>
</body>
</html>
